protos/capture.proto

```diff
@@ -4,6 +4,28 @@
 
 syntax = "proto3";
 
+message CaptureOptions {
+  bool trace_context_switches = 1;
+  int32 pid = 2;
+  double sampling_rate = 3;
+
+  enum UnwindingMethod {
+    kInvalid = 0;
+    kFramePointers = 1;
+    kDwarf = 2;
+  }
+  UnwindingMethod unwinding_method = 4;
+
+  message InstrumentedFunction {
+    string binary_path = 1;
+    uint64 file_offset = 2;
+    uint64 virtual_address = 3;
```

Comment:
is this an absolute address or vaddr in terms of the elf-file?

Reply:
Renamed to `absolute_address`.